### PR TITLE
Add tests for updating & deleting collections and categories

### PR DESCRIPTION
**Pull request by karola312**
I want to merge this change because I created tests for updating/deleting categories and collections 

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://master.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-5005 Add test for updating and deleting categories & collections](https://app.clickup.com/t/2549495/SALEOR-5005)

File: cypress/integration/catalog/categories.js [hunk 1]
```diff
@@ -136,5 +139,38 @@ filterTests({ definedTags: ["all"] }, () => {
       cy.contains(SHARED_ELEMENTS.tableRow, category.name).click();
       cy.contains(SHARED_ELEMENTS.header, category.name).should("be.visible");
     });
+
+    it("should delete category", () => {
+      const categoryName = `${startsWith}${faker.datatype.number()}`;
```
Comment (eCzerniak) [inline cypress/integration/catalog/categories.js:144]: Add an empty space.

File: cypress/integration/catalog/categories.js [hunk 2]
```diff
@@ -136,5 +139,38 @@ filterTests({ definedTags: ["all"] }, () => {
...
+          .get(BUTTON_SELECTORS.submit)
+          .click()
+          .waitForRequestAndCheckIfNoErrors("@CategoryDelete");
+        getCategory(categoryResp.id).should("be.null");
+      });
+    });
+
+    it("should update category", () => {
+      const categoryName = `${startsWith}${faker.datatype.number()}`;
+      const updatedName = `${startsWith}updatedCategory`;
```
Comment (eCzerniak) [inline cypress/integration/catalog/categories.js:159]: Add an empty space.

File: cypress/integration/catalog/collections.js [hunk 3]
```diff
@@ -182,5 +197,40 @@ filterTests({ definedTags: ["all"] }, () => {
           expect(isVisible).to.equal(false);
         });
     });
+
+    it("should delete collection", () => {
+      const collectionName = `${startsWith}${faker.datatype.number()}`;
```
Comment (eCzerniak) [inline cypress/integration/catalog/collections.js:202]: Add an empty space.

File: cypress/integration/catalog/collections.js [hunk 4]
```diff
@@ -182,5 +197,40 @@ filterTests({ definedTags: ["all"] }, () => {
...
+          .waitForRequestAndCheckIfNoErrors("@RemoveCollection");
+        getCollection({ collectionId: collectionResp.id, auth: "auth" })
+          .its("collection")
+          .should("be.null");
+      });
+    });
+
+    it("should update collection", () => {
+      const collectionName = `${startsWith}${faker.datatype.number()}`;
+      const updatedName = `${startsWith}updatedCollection`;
```
Comment (eCzerniak) [inline cypress/integration/catalog/collections.js:219]: Add an empty space.

**Review by eCzerniak — CHANGES_REQUESTED** [review]
image
image

Create category and add subcategory are failing.

**Review by Cloud11PL — APPROVED** [review]

**Review by eCzerniak — APPROVED** [review]

**Pull request merged**